### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#166] [Esercizi - Bertini]
![Screenshot 2023-10-30 171354](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/b7f366dc-d8d7-4e09-9b34-fa637fbe588e)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/61529ab4-83a0-4b00-aeb5-26d2049f9cea)

**Comment by Elia-Belli** [comment]
# Soluzione del prof
Abbiamo una moneta con truccatura $p$ incognita:
- **(i)** Lancio la moneta $n$ volte e osservo $T$ uscire $k$ volte, come decido $p$?

**Criterio di Massima Verosomiglianza**: tra tutti i possibili valori di $p$ scelgo quello che massimizza la probabilità dell'evento osservato, ovvero lo **stimatore di massima verosomiglianza** $\bar p$ per $p$ è tale che $\mathbb{P}(k\ teste\ in\ n\ lanci)$ è massima.
$$\mathbb{P}(k\ teste\ in\ n\ lanci)=\binom{n}{k}p^k(1-p)^{n-k}:=F_{n,k}(p)$$
$n$ e $k$ sono noti e $p$ incognito, trovare $\bar p=\bar p(n,k): F_{n,k}(\bar p)=max\ F_{n,k}(p)$

$$\frac{d}{dp}F_{n,k}(p)=\binom{n}{k}kp^{k-1}(1-p)^{n-k}-\binom{n}{k}p^k(n-k)(1-p)^{n-k-1}$$

$$\frac{d}{dp}F_{n,k}(p)=0 \iff \binom{n}{k}p^{k-1}(1-p)^{n-k-1}[k(1-p)-(n-k)p]=0\iff$$

$$^{p\in(0,1)}\iff k-kp-np+kp=0\iff k=np\iff p=\frac{k}{n}$$
Concludo (omettendo un po' di analisi aggiuntiva) che $\bar p(k,n)=k/n$

---
- **(ii)** Lancio la moneta finchè non rende $T$ al lancio $h$.
$$\mathbb{P}(prima\ T\ al\ lancio\ h)=\mathbb{P}(primi\ n-1\ C\ e\ ultimo\ T)=(1-p)^{n-1}p=: F_h(p)$$
$h$ noto e $p$ incognito

$$\frac{d}{dp} F_h(p)=-(h-1)(1-p)^{h-2}p+(1-p)^{h-1}=(1-p)^{h-2}[-(n-1)p+1-p]$$

$$\frac{d}{dp} F_h(p)=0 \iff^{p\in (0,1)} -hp+p+1-p=0\iff -hp+1=0 \iff p=\frac{1}{h}$$

Concludo che $\bar p(h)=1/h$
